app/controllers/answers_controller.rb

```diff
@@ -1,23 +1,39 @@
 class AnswersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :load_question, only: [:new, :create, :update, :destroy]
 
   def new
     @answer = Answer.new
   end
 
   def create
-    @question = Question.find(params[:question_id])
     @answer = @question.answers.create(answer_params)
+    @answer.user = current_user
     if @answer.save
+      flash[:notice] = 'Your answer added'
       redirect_to @question
     else
       render :new
     end
   end
 
+  def destroy
+    @answer = Answer.find(params[:id])
+    if @answer.user_id == current_user.id
+      @answer.destroy ? flash[:alert] = 'Comment deleted. RIP!' : flash[:alert] = 'We got a problem'
```

Comment:
`flash[:alert] = @answer.destroy ? 'Comment deleted. RIP!' : 'We got a problem'`